examples/README.md

```diff
@@ -203,9 +204,42 @@ For this example the following is needed in the selected region:
 * Compute Engine API: Resource policies: **one for each job in parallel** -
   _only needed for `compute` partition_
 
-[slurm-gcp-v5-cluster.yaml]: ../community/examples/slurm-gcp-v5-cluster.yaml
+[slurm-gcp-v5-centos7.yaml]: ../community/examples/slurm-gcp-v5-centos7.yaml
 [slurm-gcp]: https://github.com/SchedMD/slurm-gcp/tree/v5.0.2
 
+### [slurm-gcp-v5-ubuntu2004.yaml] ![community-badge]
+
+Similar to the previous example, but using Ubuntu 20.04 instead of CentOS 7.
```

Comment:
I'd add a note that other images are provided by slurm-gcp and that the source_image_family can be updated to use them in this example, specifying that it's not tested or officially supported of course. 
Docs: https://github.com/SchedMD/slurm-gcp/blob/master/docs/images.md#published-image-family

```markdown
SchedMD supports
[4 base images](https://github.com/SchedMD/slurm-gcp/blob/master/docs/images.md#published-image-family)
for its terraform modules. The deployment variable `source_image_family` can be updated in this
example to use any of these or a custom image built using the
[Slurm on GCP packer template](https://github.com/SchedMD/slurm-gcp/tree/v5.0.3/packer).
Note that only ubuntu and hpc-centos7 have been tested with the HPC Toolkit, other images may require additional configuration. 
```

Reply:
good catch. Added a note.